### PR TITLE
Feature: Py3 support

### PR DESCRIPTION
**Pull request by Mr-TaylorHobbs**

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-r dist.txt
+django-jenkins
```
Comment (khornberg) [inline requirements/dev.txt]: What is this used for?

**Comment by khornberg** [comment]
Otherwise LGTM

File: requirements/dev.txt (hunk 1 above)
Comment (Mr-TaylorHobbs) [inline requirements/dev.txt]: I don't know. I just moved the requirements over.

Comment (mattjmorrison-imt) [inline requirements/dev.txt]: django-jenkins can go away

**Pull request merged**